Header header
bool reference_set
homog_track/HomogMarker current_points
homog_track/HomogMarker reference_points
